validators/signup/create.js

```diff
@@ -0,0 +1,20 @@
+const Joi = require('joi');
+
+const createValidator = Joi.object({
+    firstname: Joi.string()
+        .min(3)
+        .max(30)
+        .required(),
+    lastname: Joi.string().required(),
+    age: Joi.number().required(),
+    email: Joi.string().required(),
+    date_of_birth: Joi.string().required(),
+    password: Joi.string().required(),
+    upload_photo: Joi.string().required(),
+    marita_status: Joi.string(),
+    country: Joi.string().required(),
+    job: Joi.string().required(),
+    gender: Joi.string().required()
+})
+
+exports.createValidator = createValidator
```

Comment:
since this one and the one down for update is same, why not name it user validator and reuse everywhere